airbyte-integrations/connectors/destination-redshift/build.gradle

```diff
@@ -39,7 +38,6 @@ dependencies {
 
     integrationTestJavaImplementation project(':airbyte-integrations:bases:standard-destination-test')
     integrationTestJavaImplementation project(':airbyte-integrations:connectors:destination-redshift')
-    integrationTestJavaImplementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)
```

Comment:
How do we declare the dependency of integration test on the build of the normalization image?

Reply:
Oh, nice catch. This is an oversight! I'll fix this.

Reply:
Done